doc/tutorial/pairwise_alignment/pairwise_alignment_solution_3.cpp

```diff
@@ -1,7 +1,6 @@
 #include <seqan3/alignment/pairwise/all.hpp>        // for seqan3::align_cfg and seqan3::align_pairwise
-#include <seqan3/alignment/scoring/all.hpp>         // for seqan3::aminoacid_scoring_scheme,
-                                                    //     seqan3::aminoacid_similarity_matrix, seqan3::gap_scheme,
-                                                    //     seqan3::gap_score and seqan3::gap_open_score
+#include <seqan3/alignment/scoring/all.hpp>         // for seqan3::aminoacid_scoring_scheme and
```

Comment:
I was told to instead of include <xxx.all> include single files. This speeds up compiletime. I don't know if this also applies to documentation and so on. Maybe wait for the second reviewer with that :)

Reply:
You're right for the library itself, but I believe that for the tutorials the `xxx/all.hpp` includes are preferred.